DocNet.Core/Markup/Views/assemblyViewTemplate.html

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For using the Razor Generator plugin, you'll need to add that special line at the beginning of each view (see example template).